<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="berechnen">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Berechnen</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="zuZahlenderBetrag">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.990000000005239</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="gegebenerBetrag">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.990000000005239</double>
   </property>
  </widget>
  <widget class="QLabel" name="gegebenerBetragLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gebe den gegebenen Betrag in € ein:</string>
   </property>
  </widget>
  <widget class="QLabel" name="zuZahlenderBetragLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gebe den zu zahlenden Betrag in € ein:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="rueckgeld">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rueckgeldLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dein Rückgeld beträgt in €:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZweihundertEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZweihundertEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 200€ Scheine:</string>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlEinhundertEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 100€ Scheine:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlEinhundertEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlFuenfzigEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 50€ Scheine:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZwanzigEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZwanzigEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 20€ Scheine:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlFuenfzigEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZehnEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 10€ Scheine:</string>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlEinEuroLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 1€ Münzen:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZweiEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlFuenfEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZweiEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 2€ Münzen:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlEinEuro">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlFuenfEuroLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 5€ Scheine:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZehnEuro">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZwanzigCent">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>280</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZwanzigCentLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 20c Münzen:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZehnCent">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlFuenfCent">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZweiCentLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 2c Münzen:</string>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlFuenfzigCentLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 50c Münzen:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlFuenfzigCent">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlEinCent">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlFuenfCentLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 5c Münzen:</string>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlEinCentLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>400</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 1c Münzen:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="anzahlZweiCent">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="anzahlZehnCentLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Anzahl 10c Münzen:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
